pkg/webui/components/offline/index.js

```diff
@@ -0,0 +1,59 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from '../../lib/prop-types'
+import Status from '../status'
+import Message from '../../lib/components/message'
+import sharedMessages from '../../lib/shared-messages'
+import { selectOfflineStatus } from '../../lib/selectors/offline'
```

Comment:
```suggestion
import { selectOfflineStatus } from '../../store/selectors/offline'
```
Move this to store selectors.

Reply:
As the component is in the global components folder, I thought it would be best to not only use the console store, as the import will need to be ''../../console/store/selectors/offline'', meaning device claiming will also be using the selector from the console

Reply:
Fair enough. That makes me realize also that the component needs to be in `pkg/webui/containers/`, since it is a container, connecting to the store. Let's keep the selector in `/lib` then.